### PR TITLE
Catch psql errors

### PR DESCRIPTION
**Pull request by JaxTurboNerd**
On the server side, set response status = 598 when a unique constraint error exists.  Will check for this status on the fetch call and react with a message on the pop up modal.

### CODE REVIEW

**Comment by JaxTurboNerd** [comment]
Don't merge

File: client/src/components/addName.js
Comment (murjax) [inline client/src/components/addName.js:44]: Switch this back before merging.

File: client/src/components/addName.js
Comment (murjax) [inline client/src/components/addName.js:53]: Drop this console log comment.

File: client/src/components/main.js
Comment (murjax) [inline client/src/components/main.js:10]: Switch this back too.

File: server/queries.js
Comment (murjax) [inline server/queries.js:46]: Why 598? This appears to be a network timeout code. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#598

I usually see 422 used as the response status for validation issues. Any chance of using that or are there any issues with that?

**Review by murjax — COMMENTED** [review]
Few comments, but glad to see some better handling of the constraint errors!

File: client/src/components/addName.js
Comment (JaxTurboNerd) [reply client/src/components/addName.js:44]: I will switch back.  I was using my local docker container to test these changes.

File: client/src/components/addName.js
Comment (JaxTurboNerd) [reply client/src/components/addName.js:53]: done!

File: server/queries.js
Comment (JaxTurboNerd) [reply server/queries.js:46]: changed to 422.  I just picked a random response status code for testing.

**Review by murjax — APPROVED** [review]
Good to go!

**Pull request merged**
